Taskfile.yaml

```diff
@@ -2,10 +2,88 @@
 
 version: '3'
 
+vars:
+  PKG: "./..."
+  FAILFAST: "-failfast"
+  TIMEOUT: "2m"
+  RUN: "''"
+  TAGS: "-tags json1,netgo"
+
 tasks:
+  default:
+    cmds:
+      - task: lint
+      - task: tests:local:sqlite
+      - task: tests:local:postgres
+
+  lint:
+    cmds:
+      - golangci-lint run -v --fix
+
   tests:
     cmds:
-      - go test -tags json1,netgo -v -coverpkg ./... -coverprofile coverage.out -covermode atomic ./...
+      - go test {{.TAGS}} -v -coverpkg {{.PKG}} -coverprofile coverage.out -covermode atomic {{.PKG}}
+
+  tests:local:
```

Comment:
Why local ?

Reply:
The main task run by the CI is `task tests`. It is run twice with different env variables to test both sqlite and postgres.
I added those two tasks `tests:local:sqlite` and `tests:local:postgres` to be able to run both tests locally by only running the command `task`, which also run the `lint` task, is adding colors to the results, and generate a coverage.html file.
What do you think?

Reply:
Ok, make sense.